src/libraries/System.Diagnostics.Process/tests/ProcessWaitingTests.cs

```diff
@@ -82,6 +154,45 @@ public void SingleProcess_TryWaitMultipleTimesBeforeCompleting()
             Assert.True(p.WaitForExit(0));
         }
 
+        [Fact]
+        public async Task SingleProcess_TryWaitAsyncMultipleTimesBeforeCompleting()
+        {
+            Process p = CreateProcessLong();
+            p.Start();
+
+            // Verify we can try to wait for the process to exit multiple times
+            using (var cts = new CancellationTokenSource(0))
```

Comment:
```suggestion
            using (var cts = new CancellationTokenSource(millisecondsDelay: 0))
```

avoid numbers without an obvious meaning

Reply:
Fixed!